core/src/main/java/com/amplifyframework/geo/models/BoundingBox.java

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
Can we add new classes in Kotlin?

Reply:
this is being added in `core` library which is written in Java. Our Kotlin policy is to write any new module in Kotlin while keeping Java modules purely Java.